source/Table/Table.js

```diff
@@ -315,7 +320,7 @@ export default class Table extends PureComponent {
           autoContainerWidth
           className={cn('ReactVirtualized__Table__Grid', gridClassName)}
           cellRenderer={this._createRow}
-          columnWidth={width}
+          columnWidth={totalColumnWidth}
```

Comment:
Otherwise tables may end up looking like the demo site does now (with this branch):
<img width="551" alt="screen shot 2017-03-04 at 9 37 21 am" src="https://cloud.githubusercontent.com/assets/29597/23580848/38821fc4-00be-11e7-9f63-ded708a9126b.png">